src/azure-cli/azure/cli/command_modules/acr/helm.py

```diff
@@ -175,6 +182,81 @@ def acr_helm_repo_add(cmd,
     p.wait()
 
 
+def acr_helm_install_cli(client_version='2.16.3', install_location=None, yes=False):
+    """Install Helm command-line tool."""
+
+    if client_version >= '3':
+        raise CLIError('Helm v3 is not supported yet.')
```

Comment:
Please warn people that it is not we don't support helm3, but just can't get along with "az acr helm" commands. 
Maybe output a warning: 
```
Please note helm3 doesn't work with "az acr helm" commands, and but you can still push helm chat using a different command flow to ACR. For more information, please check out  https://docs.microsoft.com/en-us/azure/container-registry/container-registry-helm-repos